src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -808,7 +808,7 @@ private void changeSchematic()
      * @param complete if pasted, should it be complete.
      * @param structureName of the scan to be built.
      */
-    private static void requestScannedSchematic(@NotNull final StructureName structureName, final boolean paste, final boolean complete)
+    public static void requestScannedSchematic(@NotNull final StructureName structureName, final boolean paste, final boolean complete)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 30 to the 20 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)